cmd/earthly/subcmd/build_cmd.go

```diff
@@ -132,22 +132,22 @@ func (a *Build) Action(cliCtx *cli.Context) error {
 		a.cli.Flags().Strict = true
 
 		if a.cli.Flags().InteractiveDebugging {
-			return errors.New("unable to use --ci flag in combination with --interactive flag")
+			return params.Errorf("unable to use --ci flag in combination with --interactive flag")
```

Comment:
It seems that we only cover the build command here. Should we now perhaps change all other commands too for consistency? That might be a bigger change.

Reply:
yes, I agree, I decided to make that change in a separate PR.
However, the difference for the user wouldn't be substantial anyway - 
Errors would be printed the same (or very similar), we just wouldn't be sending the new logstream failure enum type and wrapped messages would show the full error stack.